OMCompiler/SimulationRuntime/fmi/export/openmodelica/fmu_read_flags.c.inc

```diff
@@ -192,6 +192,13 @@ int FMI2CS_initializeSolverData(ModelInstance* comp)
     solverInfo->solverMethod = S_EULER;
   }
 
+  /*  If no states are present use Euler's method since nothing is happening anyway. */
+  if (data->modelData->nStates < 1)
+  {
+    FILTERED_LOG(comp, fmi2OK, LOG_ALL, "fmi2Instantiate: No states present, using eulers method.")
```

Comment:
This doesn't explain too much to inexperienced users and might be confusing. Shouldn't we just introduce an algebraic mode that we trigger instead to not mix the solvers?

Reply:
Well it is roughly the same message users will get with the C runtime.
What do you mean by algebraic mode? Just evaluate the right hand side of the ODE instead of using any solver at all?

Reply:
It is _not_ an ODE system which makes it an odd message.

Reply:
I agree that we should not use an ODE solver for a simple equation system without any states. But that is what the C runtime is currently doing. Basically it is only updating time and algebraic variables.
I can change the message for the C runtime and FMI to something like "No states present, continuing without ODE solver."